inlong-dashboard/src/ui/locales/cn.json

```diff
@@ -362,7 +363,7 @@
   "meta.Sinks.Kudu.FieldDescription": "描述",
   "meta.Sinks.Kudu.buckets": "Buckets",
   "meta.Group.InlongGroupId": "数据流组 ID",
-  "meta.Group.InlongGroupIdRules": "只能包含小写字母、数字、中划线、下划线",
+  "meta.Group.InlongGroupIdRules": "只能包含英文字母、数字、中划线、下划线",
```

Comment:
**Other Chinese and English descriptions are also suggested to be modified in this way.**

```suggestion
  "meta.Group.InlongGroupIdRules": "只能包含英文字母、数字、点号(.)、中划线(-)、下划线(_)",
```